openmc/capi/plot.py

```diff
@@ -218,3 +218,27 @@ def id_map(plot):
     _dll.openmc_id_map(POINTER(_PlotBase)(plot),
                        img_data.ctypes.data_as(POINTER(c_int32)))
     return img_data
+
+
+def property_map(plot):
+    """
+    Generate a 2-D map of (cell_temperature, material_density). Used for
```

Comment:
"cell_temperature" and "material_density" don't have any particular meaning here, so I would just use English: "Generate a 2-D map of cell temperature and material density."